.github/workflows/deploy_collab.yml

```diff
@@ -14,8 +14,7 @@ jobs:
   style:
     name: Check formatting and Clippy lints
     runs-on:
-      - self-hosted
-      - test
+      - buildjet-16vcpu-ubuntu-2204
```

Comment:
I think this job currently runs on macOS, so we can probably keep it there.

AFAICT that `self-hosted` and `test` label pair only corresponds to the macOS runners.